t/obs/openSUSE:Factory:ARM:ToTest/base/print_rsync_repo.before

```diff
@@ -1,16 +1,8 @@
-rsync --timeout=3600 -rtlp4 --delete --specials --include=java* --include=kernel-default-debug* --include=kernel-default-base-debug* --include=mraa-debug* --include=wicked-debug* --exclude=aarch64/* --exclude=armv7hl/* --exclude=i586/* --exclude=i686/* --exclude=noarch/* --exclude=nosrc/* --exclude=ppc64/* --exclude=ppc64le/* --exclude=s390x/* --exclude=src/* --exclude=x86_64/* obspublish::openqa/openSUSE:Factory:ARM:ToTest/*images/local/*openSUSE-ftp-ftp*armv7hl*/*Media2/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-CURRENT-debuginfo/
-rsync --timeout=3600 -rtlp4 --delete --specials --link-dest /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-CURRENT-debuginfo/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-CURRENT-debuginfo/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610-debuginfo
-rsync --timeout=3600 -rtlp4 --delete --specials --include=coreutils* --include=yast2-network* --exclude=aarch64/* --exclude=armv7hl/* --exclude=i586/* --exclude=i686/* --exclude=noarch/* --exclude=nosrc/* --exclude=ppc64/* --exclude=ppc64le/* --exclude=s390x/* --exclude=src/* --exclude=x86_64/* obspublish::openqa/openSUSE:Factory:ARM:ToTest/*images/local/*openSUSE-ftp-ftp*aarch64*/*Media3/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-aarch64-CURRENT-source/
-rsync --timeout=3600 -rtlp4 --delete --specials --link-dest /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-aarch64-CURRENT-source/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-aarch64-CURRENT-source/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-aarch64-Snapshot20240610-source
-rsync --timeout=3600 -rtlp4 --delete --specials --include=coreutils* --include=yast2-network* --exclude=aarch64/* --exclude=armv7hl/* --exclude=i586/* --exclude=i686/* --exclude=noarch/* --exclude=nosrc/* --exclude=ppc64/* --exclude=ppc64le/* --exclude=s390x/* --exclude=src/* --exclude=x86_64/* obspublish::openqa/openSUSE:Factory:ARM:ToTest/*images/local/*openSUSE-ftp-ftp*armv7hl*/*Media3/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-CURRENT-source/
-rsync --timeout=3600 -rtlp4 --delete --specials --link-dest /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-CURRENT-source/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-CURRENT-source/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610-source
+rsync --timeout=3600 -rtlp4 --delete --specials obspublish::openqa/openSUSE:Factory:ARM:ToTest/images/local/0openSUSE/openSUSE-20250730-armv6hl-armv7hl-aarch64/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss--CURRENT/
+rsync --timeout=3600 -rtlp4 --delete --specials --link-dest /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss--CURRENT/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss--CURRENT/ /var/lib/openqa/factory/repo/openSUSE-Tumbleweed-oss--Snapshot20240610
```

Comment:
we got empty ARCH between oss--CURRENT

Reply:
Good catch, sadly it breaks and using the multiarch=1 trick didn't work